lib/need_for_type/states/menu_state.rb

```diff
@@ -0,0 +1,20 @@
+module NeedForType::States
+  class MenuState < State
+
+    def initialize(display_window)
+      super(display_window)
+    end
+
+    def input_worker(number_option)
```

Comment:
`number_option` refers to the option being handled right now perhaps we can change it to `option_number`